web/styles/app_components.css

```diff
@@ -306,9 +302,8 @@ input::placeholder {
 
             .ind-tab {
                 display: flex;
-                text-overflow: initial;
-                white-space: initial;
-                vertical-align: middle;
+                min-width: 0;
```

Comment:
While I cannot find any bugs due to these CSS changes, it would be better if you could avoid these changes here and add the required changes for tabs used in user section only like we do for other cases so that we do not have any unnoticed bug. If we think that these CSS rules are unnecessary and can be updated, we can do so in a separate PR.

Reply:
I understand the concern. I have updated the required changes.
Though , it would be a great idea to update the ```tab-switcher``` component to increase it's width according to the text rather than defining the concrete-width for every class wherever required.

Probably there should be a separate issue/PR to update these changes.

Reply:
Yeah that seems like a good potential follow-up PR.

Reply:
Sure Tim , I will immediately create a new follow-up PR once this is succesfully reviewed.